src/components/FinalPoem.js

```diff
@@ -3,23 +3,36 @@ import PropTypes from 'prop-types';
 import './FinalPoem.css';
 
 const FinalPoem = (props) => {
```

Comment:
Just a note that it looks like the tests are failing because you used different props like `isRevealed` instead of `isSubmitted`.  That's why it looks like the tests are failing.